src/Network/Network/ChangeLog.md

```diff
@@ -74,6 +79,9 @@
     - `Update-AzVirtualHub`
 * Updated cmdlets to expose two read-only properties of client certificate.
     - `Get-AzApplicationGatewayTrustedClientCertificate`
+* Updated below cmdlets to enable or disable BgpRouteTranslationForNat on VpnGateway.
```

Comment:
Is this should move to `Upcoming Release`?

Reply:
Moved it under Upcoming Release section.